test/integration/patients/worklist/worklist.js

```diff
@@ -2137,6 +2137,38 @@ context('worklist page', function() {
     cy
       .get('@filtersSidebar')
       .should('not.exist');
+
+    cy
+      .get('.list-page__filters')
+      .find('[data-all-filters-region]')
+      .click();
+
+    cy
+      .get('[data-select-all-region]')
+      .click();
+
+    cy
+      .get('@filtersSidebar')
+      .find('.js-close')
+      .click();
+
+    cy
+      .get('@filtersSidebar')
+      .should('not.exist');
+
+    cy
+      .get('[data-filters-region]')
+      .find('.js-cancel')
+      .click();
+
+    cy
+      .get('.list-page__filters')
+      .find('[data-all-filters-region]')
+      .click();
+
+    cy
+      .get('@filtersSidebar')
+      .should('exist');
```

Comment:
Added this to the test:

1. Open the filters sidebar.
2. Select all list items (starts bulk select app).
3. Cancel select all by clicking the `Cancel` option.
4. Verify the filters sidebar can be closed and then opened properly after that.